src/main/java/com/zombiecastlerush/building/Room.java

```diff
@@ -37,18 +38,26 @@ public void setDescription(String description) {
         return connectedRooms;
     }
 
-    public Inventory getInventory() {
-        return inventory;
+    public Challenge getChallenge() {
+        return challenge;
+    }
+
+    public void setChallenge(Challenge challenge) {
+        this.challenge = challenge;
     }
 
     @Override
-    public String toString() {
+    public String toString(){
         return getName();//+ "Connected Rooms: " + connectedRooms.toString(;
     }
 
     //Methods
     //add room to the connected rooms List for this room
-    public void addConnectedRooms(Room room) {
-        this.connectedRooms.add(room);
+    public void addConnectedRooms(Room ...rooms) {
+        for(Room room:rooms){
+            this.connectedRooms.add(room);
+        }
+
```

Comment:
how do we determine two Rooms are the same or not?

Reply:
keep unique names for rooms for now